### ISSUE TITLE
Maintaining this repo

### ISSUE THREAD

**Issue by yaron2** [tjwp/dapr-ruby-sdk#4]
Hi there,

Dapr maintainer here.
Would you be interested in moving (and maintaining) this repository to the `dapr` org?

**Comment by tjwp** [comment]
Yes, I'd be happy to! Let me know if there are any changes or updates that you require before moving it to the `dapr` org -- a pointer to any existing doc is fine. Thanks!

**Comment by yaron2** [comment]
Are you on our Discord channel? if not, please join and send me a friend request. my handle is `yaron2`. We can take it from there

**Comment by tjwp** [comment]
Invite sent from handle `tjwp`, thanks!

**Comment by exegeteio** [comment]
Any updates on this transition?

**Comment by philipbjorge** [comment]
@yaron2 @tjwp --
Any updates on this transition?

**Comment by tjwp** [comment]
No news on this on my end. I can re-join the Discord and reach out again.

**Comment by taiyyu-coupa** [comment]
@yaron2 @tjwp
Can we update to Dapr 1.10 proto files, ruby and Gems?
I did that without running tests w/ this PR:
https://github.com/tjwp/dapr-ruby-sdk/pull/6/files

**Comment by smshuja** [comment]
@yaron2 It would be great if there is an official version of ruby sdk.
